src/quo2/screens/info/page_nav.cljs

```diff
@@ -0,0 +1,114 @@
+                                                                                     :mid-section-type     :text-with-one-icon})
+                            :one-icon-align-right?     (create-variation base-props {:one-icon-align-left? false
+                                                                                     :mid-section-type     :text-with-one-icon})
+                            :two-icons?                (create-variation base-props {:mid-section-type :text-with-two-icons})
+                            :user-icon?                (create-variation base-props {:align-mid?              true
+                                                                                     :horizontal-description? false
+                                                                                     :mid-section-type        :text-with-one-icon})
+                            :empty?                    (create-variation base-props {:mid-section-main-text   ""
+                                                                                     :mid-section-description ""})
+                            :align-left-with-icon?     (create-variation base-props {:align-mid?       true
+                                                                                     :mid-section-type :text-with-one-icon})
+                            :align-left-top-down-text? (create-variation base-props {:align-mid?              true
+                                                                                     :horizontal-description? false
+                                                                                     :mid-section-type        :text-with-description})}
+        state              (reagent/atom (-> (get variations (:selected-variation @selected-variation))
+                                             (assoc :right-section-icons [right-icon])))]
+    (fn []
+      (let [too-many-right-icons? (-> (:right-section-icons @state)
+                                      count
+                                      (<= 2))]
+        [rn/view {:margin-bottom 50
+                  :padding       16}
+         [rn/view {:flex 1}
+          [preview/customizer selected-variation descriptor]]
+         [rn/view {:padding-vertical 30
+                   :flex-direction   :row
+                   :justify-content  :center}
+          [quo2/page-nav @state]]
+         [rn/touchable-opacity {:style    {:width  150
+                                           :height 50}
+                                :on-press (fn [_]
+                                            (when too-many-right-icons?
+                                              (swap! state update-in [:right-section-icons] conj right-icon)))}
+          [rn/text {:style {:color (if (theme/dark?)
+                                     colors/white
+                                     colors/neutral-95)}}
+           (if too-many-right-icons?
+             "Add right icon"
```

Comment:
consider removing this and add a select in the descriptor with options 1,2 and 3. It makes it easier to interact with, otherwise you have to go on and off the page to reset it.

Reply:
Done